samples/matter/light-switch/src/ZclCallbacks.cpp

```diff
@@ -0,0 +1,36 @@
+/* Copyright (C) 2024 Alif Semiconductor - All Rights Reserved.
```

Comment:
```suggestion
/* Copyright (C) 2025 Alif Semiconductor - All Rights Reserved.
```

Reply:
I will fix this to Next PR when I create a documentation to samples.